config/peerpods/podvm/lib.sh

```diff
@@ -244,6 +244,18 @@ function prepare_source_code() {
         fi
     fi
 
+    # Enable image signature check
+    if [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]]; then
+	cat<<EOF>"${podvm_dir}"/files/etc/agent-config.toml
+server_addr = "unix:///run/kata-containers/agent.sock"
+guest_components_procs = "none"
+image_registry_auth = "file:///run/peerpod/auth.json"
+enable_signature_verification = true
```

Comment:
What happens if image_policy_file doesn't exist? will it fail? should we allow some variable in the podvm-image-cm to skip signature check?

Reply:
IIUC agent will crash if the KBS doesn't have the policy. It is up to the KBS _owner_ to make sure there's a default container policy.

Reply:
Let's go without an extra knob for now as it isn't strictly needed for the upcoming release.